TooltipDecorator/Tooltip.js

```diff
@@ -0,0 +1,171 @@
+import kind from '@enact/core/kind';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Skinnable from '../Skinnable';
+import TooltipLabel from './TooltipLabel';
```

Comment:
We prefer to add the one empty line between the imported module out of the current directory and the imported module in the current directory.

```
import Skinnable from '../Skinnable';

import TooltipLabel from './TooltipLabel';
```